Tools/CISupport/ews-build/steps.py

```diff
@@ -623,14 +627,17 @@ def run(self):
         remote = self.getProperty('github.head.repo.full_name', 'origin').split('/')[0]
         project = self.getProperty('github.head.repo.full_name', self.getProperty('project'))
         branch = self.getProperty('github.head.ref', 'main')
+        base = self.getProperty('github.base.sha')
+        rebase_target = self.getProperty('ews_revision') or self.getProperty('got_revision')
 
         for command in [
```

Comment:
Nit: For better readability, let's store this in a variable (like `commands`), and then iterate over it (`for command in commands:`)